### DISCUSSION TITLE
How to interpret the zigbee2mqtt map

### DISCUSSION THREAD

**Discussion by petervflocke** [Koenkk/zigbee2mqtt#7335] [General]
First of all, a great project and "piece" of software, do appreciate your time and know-how investment!

As I'm a quite new and still in a learning curve, could you please answer or suggest further reading for the following topic.
I like the map of my current network of the zigbee devices:
![Screenshot from 2021-05-04 17-33-23](https://user-images.githubusercontent.com/23005024/117031158-9bc64680-ad00-11eb-9027-25a67048a09b.png)
It would be also helpful, if I could interpret it in a proper way. Can you please correct or extend my statements and answer few questions:

- The connection lines show the current binding and the link quality between device and router(s)/coordinator
- The "unconnected" devices (Aqara water leak Xwater01/2 and Aqara door sensor XDoor01) are somehow (how?) in the network. They can successfully report their status and are available in the HA
- Is it required to force them to be always connected?
- How to force them to be connected? Repairing is somehow awkward, since they will lose their connection again (already tested)
- Does it mean that I may need more routers, closer to the devices?

I'd appreciate any kind of your suggestion.
Best
Peter.

**Comment by Gr4ffy** [comment]
> The "unconnected" devices (Aqara water leak Xwater01/2 and Aqara door sensor XDoor01) are somehow (how?) in the network. They can successfully report their status and are available in the HA

Maybe because of this?
https://www.zigbee2mqtt.io/devices/MCCGQ11LM.html#troubleshooting-device-stops-sending-messagesdisconnects-from-network

**Reply by petervflocke** [reply]
Hi @Gr4ffy 

Thank you for your feedback. I'm aware of the general Aqqara/Xiaomi problems and do not complain about the integration. 
What I do not know is how to interpret the missing connection on the map. What the line on the map really means?
Even if the door sensor is not connected (on the map) **it works perfectly** and reports its status, signal and battery without any issue.

**Reply by Wolk9** [reply]
I do have the same experience and interpretation of the map. It doesn't correspond to my logics of the map and the reality.

**Reply by petervflocke** [reply]
Nice not to be alone, but it does not answer my questions. 
If it is a bug, then let it be, but it would be great just to know.
Anyway, thanks for joining the club ;)
